metpy/calc/basic.py

```diff
@@ -106,6 +107,12 @@ def get_wind_components(speed, wdir):
      <Quantity(7.071067811865477, 'meter / second')>)
 
     """
+    try:
+        wdir = wdir.to('radians')
+    except:
+        pass
+    if np.greater(np.nanmax(wdir), 2 * np.pi * units.radians):
```

Comment:
Can we make this 4 pi? I can see it being not-so-rare to have some values in [2pi, 4pi].